core/src/models/Property.ts

```diff
@@ -285,6 +285,8 @@ export class Property extends CommonModel<Property> {
     return this.idIsDefault() ? ConfigWriter.generateId(this.key) : this.id;
   }
 
+  uniqueIdentifier = ["key", "sourceId"];
```

Comment:
It was a little strange to me that the old version didn't need to check the source:key uniqueness, but this was necessary to make the app run under this new setup.